### ISSUE TITLE
Steps missing a notification of some sort

### ISSUE THREAD

**Issue by thiessenp-cds** [cds-snc/platform-forms-client#6408]
Navigating between the steps is not clear to AT users. The page title could be updated along with focussing the sub heading.
